lib/modules/dosomething/dosomething_kudos/dosomething_kudos.module

```diff
@@ -131,67 +126,120 @@ function dosomething_kudos_sort_by_taxonomy_term($a, $b) {
   return $result;
 }
 
+/**
+ * Get current user from supplied data array or global user.
+ *
+ * @param  array  $data
+ * @return null|string
+ */
+function dosomething_kudos_get_current_user($data = []) {
```

Comment:
Yeah, not super keen on it, but added it to take into account if a user is passed via a URL param (which it now supports) versus using the current global user. Chatted w/ @DFurnes about this and we both kind of agreed it might just be best to remove the newly added `?current_user=120193` url param on the `/reportback-items` endpoint and just use the `global $user`. Eventually as we integrate the new OAuth2 stuff FURRRRRRRNS has worked on, we can add some logic to if no `global $user` to check against a `JSON Web Token` or whatever the OAuth-y approach is :)